mac

```diff
@@ -150,6 +150,8 @@ brew update
 
 brew_install_or_upgrade 'zsh'
 brew_install_or_upgrade 'git'
+brew_install_or_upgrade 'mongodb'
+brew_launchctl_restart 'mongodb'
```

Comment:
Will move this (and Alfred, and Dash) to the laptop.local section as possible/suggested customizations to the script.